fips/views.py

```diff
@@ -0,0 +1,21 @@
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+from .models import FipsEntity
+from .serializers import FipsSerializer
+
+
+@api_view(['GET'])
+def search_view(request):
+    """
+    Search for a location.
+    ---
+    parameters:
+      - name: q
+        description: The user's search query
+        type: string
+        paramType: query
+    """
+    query = request.query_params.get('q', '').lower()
+    query_set = FipsEntity.objects.filter(name__icontains=query)
```

Comment:
Thoughts on `_icontains` vs. `_istartswith`)? Probably most people won't try to find San Diego by starting with Diego. I think.